core/src/Select/Select.less

```diff
@@ -64,11 +65,11 @@
       display: block;
       flex: 1;
       overflow: hidden;
-      padding: 8px 12px;
+      padding: 7px 8px 6px 10px;
```

Comment:
These values have been tuned quite a bit across multiple products and views. Ensure that you are confident in these changes and they do not contain product wide unintended breaking styling.

Reply:
Change lines the size up to the buttons, makes text smaller for more breathing room and space for characters to show
![image](https://user-images.githubusercontent.com/729972/61345648-e5336200-a80a-11e9-894d-492684e7def7.png)
![image](https://user-images.githubusercontent.com/729972/61345750-2f1c4800-a80b-11e9-9bcb-d778590286ff.png)
![image](https://user-images.githubusercontent.com/729972/61396481-adfd9900-a87c-11e9-81b1-ac475b76a6e1.png)